src/model/openthaigpt_pretraining_model/llama_thai_tokenizer/merge_tokenizer.py

```diff
@@ -0,0 +1,52 @@
+from transformers import LlamaTokenizer
+from sentencepiece import sentencepiece_model_pb2 as sp_pb2_model
+import sentencepiece as spm
+from openthaigpt_pretraining_model.llama_thai_tokenizer.constants import (
+    LLAMA_TOKENIZER_DIR,
+    THAI_SP_MODEL_DIR,
+    # OUTPUT_SP_DIR,
+    # OUTPUT_HF_DIR,
+)
+
+
+def merge():
+    llama_tokenizer = LlamaTokenizer.from_pretrained(LLAMA_TOKENIZER_DIR)
+    thai_sp_model = spm.SentencePieceProcessor()
+    thai_sp_model.Load(THAI_SP_MODEL_DIR)
+
+    llama_spm = sp_pb2_model.ModelProto()
+    llama_spm.ParseFromString(llama_tokenizer.sp_model.serialized_model_proto())
+    thai_spm = sp_pb2_model.ModelProto()
+    thai_spm.ParseFromString(thai_sp_model.serialized_model_proto())
+
+    llama_spm_tokens = {p.piece for p in llama_spm.pieces}
+    # check before merge vocabs
+    # print(f"Before: {len(llama_spm_tokens)}")
+
+    for p in thai_spm.pieces:
+        piece = p.piece
+        if piece not in llama_spm_tokens:
+            new_p = sp_pb2_model.ModelProto().SentencePiece()
+            new_p.piece = piece
+            new_p.score = 0.0
+            llama_spm.pieces.append(new_p)
+
+    # check after merge vocabs
+    # llama_spm_tokens2 = {p.piece for p in llama_spm.pieces}
+    # print(f"After: {len(llama_spm_tokens2)}")
+
+    llama_tokenizer.sp_model = spm.SentencePieceProcessor(
+        model_proto=llama_spm.SerializeToString()
+    )
+    return llama_tokenizer
+
+
+# to save SP model : uncomment following lines
```

Comment:
Comment like this is not looking good in our code.
Can you split this file into two files?
1. llama_thai_tokenizer/merge.py has
-  `def merge():`
3. scripts/merge_tokenizer.py has
- `from openthai_gpt_pretraining_model.llama_thai_tokenizer.merge import merge` 
- `tokenizer.save_pretrained(OUTPUT_HF_DIR)`

Reply:
done